test/crawlers/puppeteer_crawler.test.js

```diff
@@ -154,7 +176,8 @@ describe('PuppeteerCrawler', () => {
         expect.hasAssertions();
     });
 
-    test('supports useChrome option', async () => {
+    // FIXME: I have no idea why but this test hangs
+    test.skip('supports useChrome option', async () => {
```

Comment:
Like always or after the changes in this PR?

Reply:
After the changes. It hangs locally, haven't tried the CI.

Reply:
Hmm, maybe the proxy per context is broken in Chrome with Puppeteer? Would be worth investigating. Puppeteer does not officially support Chrome.

Reply:
Tested again without the changes and it hangs as well.

Reply:
I mean jest says all tests passed but the process never exits. If I skip the test, it exits as expected.

Reply:
ok